librz/core/cmd/cmd_debug.c

```diff
@@ -2235,51 +2181,187 @@ RZ_IPI RzCmdStatus rz_cmd_debug_step_until_optype_handler(RzCore *core, int argc
 	return RZ_CMD_STATUS_OK;
 }
 
+// dsue
 RZ_IPI RzCmdStatus rz_cmd_debug_step_until_esil_handler(RzCore *core, int argc, const char **argv) {
 	step_until_esil(core, argv[1]);
 	rz_core_dbg_follow_seek_register(core);
 	return RZ_CMD_STATUS_OK;
 }
 
+// dsuf
 RZ_IPI RzCmdStatus rz_cmd_debug_step_until_flag_handler(RzCore *core, int argc, const char **argv) {
 	step_until_flag(core, argv[1]);
 	rz_core_dbg_follow_seek_register(core);
 	return RZ_CMD_STATUS_OK;
 }
 
+// dt
+RZ_IPI RzCmdStatus rz_cmd_debug_trace_handler(RzCore *core, int argc, const char **argv) {
+	RzDebugTracepoint *t;
+	if ((t = rz_debug_trace_get(core->dbg, core->offset))) {
+		rz_cons_printf("offset = 0x%" PFMT64x "\n", t->addr);
+		rz_cons_printf("opsize = %d\n", t->size);
+		rz_cons_printf("times = %d\n", t->times);
+		rz_cons_printf("count = %d\n", t->count);
+		// TODO cons_printf("time = %d\n", t->tm);
+	}
+	return RZ_CMD_STATUS_OK;
+}
+
+// dtl
+RZ_IPI RzCmdStatus rz_cmd_debug_traces_handler(RzCore *core, int argc, const char **argv, RzCmdStateOutput *state) {
+	rz_debug_trace_print(core->dbg, state->mode, core->offset);
+	return RZ_CMD_STATUS_OK;
+}
+
+// dtl=
+RZ_IPI RzCmdStatus rz_cmd_debug_traces_ascii_handler(RzCore *core, int argc, const char **argv) {
+	rz_debug_traces_ascii(core->dbg, core->offset);
+	return RZ_CMD_STATUS_OK;
+}
+
+// dt+
+RZ_IPI RzCmdStatus rz_cmd_debug_trace_add_handler(RzCore *core, int argc, const char **argv) {
+	int count = argc > 2 ? rz_num_math(core->num, argv[1]) : 1;
```

Comment:
```suggestion
	int count = argc > 1 ? rz_num_math(core->num, argv[1]) : 1;
```
Maybe worth adding a test for this command, considering this was not detected by CI.

Reply:
We have very low coverage for debug code unfortunately.